intercom-plugin/src/ios/IntercomBridge.m

```diff
@@ -114,8 +117,66 @@ - (void)displayHelpCenter:(CDVInvokedUrlCommand*)command {
     [self sendSuccess:command];
 }
 
-- (void)hideMessenger:(CDVInvokedUrlCommand*)command {
-    [Intercom hideMessenger];
+- (void)displayHelpCenterCollections:(CDVInvokedUrlCommand*)command {
+    NSDictionary *args = command.arguments[0];
+    NSArray* collectionIds = args[@"collectionIds"];
+    [Intercom presentHelpCenterCollections:collectionIds];
+    [self sendSuccess:command];
+}
+
+- (void)fetchHelpCenterCollections:(CDVInvokedUrlCommand*)command {
+    [Intercom fetchHelpCenterCollectionsWithCompletion:^(NSArray<ICMHelpCenterCollection *> * _Nullable collections, NSError * _Nullable error) {
+        if (error) {
+            CDVPluginResult *pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_ERROR messageAsNSInteger:error.code];
+            [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
+        } else {
+            NSMutableArray *array = [[NSMutableArray alloc] init];
+            for (int i = 0; i < collections.count; i++)
+            {
+                [array addObject:collections[i].toDictionary];
```

Comment:
While I'm being difficult, can we change the `object.toDictionary` calls to `[object toDictionary]`. 
What you have is legal and valid. But it's not the done thing and feels deceiving to an Objective-C person coming in wondering why something that looks property is reading like a verb. 
[This person put it more bluntly... using dot syntax here is evil](https://stackoverflow.com/a/2375991/11807742) 😜 😂

Reply:
😮  this was a nice TIL. 
I've changed it now!